src/sentry/static/sentry/app/views/releases/list/organizationReleases/index.jsx

```diff
@@ -54,21 +55,34 @@ class OrganizationReleases extends AsyncView {
     });
   };
 
+  hasFilters() {
+    const {location} = this.props;
+
+    const hasQueryFilter = !!location.query.query;
+    const hasEnvironmentFilter = !!location.query.environment;
+
+    // If all or one project is selected show the setup screen
+    // We check the type since a single project value is represented as a
+    // string when deserialized and multiple values as an array.
+    const hasProjectFilter =
+      !location.query.project || Array.isArray(location.query.project);
```

Comment:
Is it better to read from the query string or from the GlobalSelectionStore/selection props forwarded by `withGlobalSelection`?

Reply:
I used it in this case since we'll also need `cursor` from here, the events and user feedback pages also use the value from the URL